### PR TITLE
[Dominion] Video Modal

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Update video modal look.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Screen Shot 2019-03-12 at 2 38 39 PM

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by nerdlet** [comment]
@karimkawambwa Looks good to me. Maybe a heads up for later, used the same modal events for `How it Works` Component link to the video list...I'll create a story update the code on my end

**Comment by karimkawambwa** [comment]
> Maybe a heads up for later

Will do pings on the dominion group but this change should not affect behaviour on your code, its none breaking but will definitely need update.

File: dominion/dominion_ui/src/components/Video/Thumbnail.js [hunk 1]
```diff
@@ -47,10 +47,11 @@ function Thumbnail({ classes, isSelected, onClick, videoId, videoTitle }) {
   return (
     <div
       style={{
-        height: 100,
-        backgroundImage: `url(https://img.youtube.com/vi/${videoId}/default.jpg)`,
+        backgroundImage: `url(https://img.youtube.com/vi/${videoId}/sddefault.jpg)`,
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/Video/Thumbnail.js]: `sddefault` is not always available

File: dominion/dominion_ui/src/components/Video/Player.js [hunk 2]
```diff
@@ -58,32 +62,29 @@ class Player extends React.Component {
...
+                src={`https://www.youtube-nocookie.com/embed/${videoId}`}
+                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              />
+            </div>
+          </Grid>
+
+          <Grid item>
             <Grid
-              container
+              item
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/Video/Player.js:102]: Only `<Grid container` or `<Grid item container` can contain `direction`, etc.
```
Warning: Failed prop type: The property `direction` of `Grid` must be used on `container`.
    in WithStyles(Grid) (at Player.js:82)
```

**Review by kilemensi — CHANGES_REQUESTED** [review]
@karimkawambwa:

 1. `Modal` component should encapsulate the `Navigation` logic. I see we're still moving `Navigation` around with the `Modal`. The only thing other components need to do is open and say what type of modal to open e.g. video, search, etc.,
 2. Scrollbars (especially horizontal ones) looks bad and should be removed,
Screenshot from 2019-03-13 15-41-11
 3. Mobile design needs improvement. It's quite difficult to tell the current video from playlist. I think playlist needs to be further down from current video and the thumbnails need to be _a wee bit smaller_ than the video (may be 75-85% of the screen width, but not 100%), and
Screenshot from 2019-03-13 15-54-16
 4. Clicking anywhere on the thumbnail should switch video (not just play button).

**Comment by karimkawambwa** [comment]
> Modal component should encapsulate the Navigation logic. I see we're still moving Navigation around with the Modal. The only thing other components need to do is open and say what type of modal to open e.g. video, search, etc.,

Could i get some thoughts on the structure because at the moment in circles. Perhaps a different eye could help make this modal better.

or could this be a different story ?

@KhadijaMahanga @nerdlet @kilemensi

**Comment by kilemensi** [comment]
Create a chore story to track the Navigation/Modal implementaion and we'll deal with it at a later stage @karimkawambwa.

Just focus on the Video model on this PR.

  * Reduce the vertical space between Navigation and Video,
  * I think we should remove the scrolls altogether since they make our implementation a bit too different from the designs. Lets limit the maximum number of videos on the playlist... something between 4-6?,
  * The Player background has a bit of transparency, and
  * Playlist video titles are a bit harder to read on our implementation (increase size?).

Design

Screenshot from 2019-03-14 11-08-47

Current implementation

Screenshot from 2019-03-14 11-09-34

**Review by KhadijaMahanga — CHANGES_REQUESTED** [review]
Also on titles, apart from increasing the font size..let's also change the font to `Lora` as per design

**Comment by karimkawambwa** [comment]
@KhadijaMahanga @kilemensi 👀

**Review by KhadijaMahanga — APPROVED** [review]

**Pull request merged**